.github/workflows/README.md

```diff
@@ -0,0 +1,67 @@
+# Toolbelt Workflow
+
+Goal: A mechanism that implements as many OpenSSF best practices using OpenSSF tools as possible in a way that is simple to use for project maintainers and easy for end users to check projects using this mechanism are following best practices and creating important security metadata (e.g. SBOMs, in-toto SLSA, etc.)
+
+Secondary Goals:
+
+- Highlight where there are gaps in OpenSSF and related practices, frameworks, and tools, or their adoption, especially in different ecosystems (e.g. Sigstore signatures are not supported in Maven)
+- Highlight when implementing these practices or adopting the tools is difficult
+Show examples following these practices and implementing the tools 
+
+Outcomes:
+
+- Maintainers should be able to use this mechanism to implement OpenSSF practices and adopt OpenSSF tooling, e.g. add SLSA to their build and turn on Scorecard.
+- Maintainers should be able to easily know where data is generated, e.g. where SBOMs, SLSA, SAST reports, etc. live so they can use it for later analysis and understanding supply chain risks.
+- End users should also be able to easily know where data is generated, e.g. where SBOMs, SLSA, SAST reports, etc. live so they can use it to make decisions on risk as well as analysis later on when vulnerabilities or supply chain incidents happen.
+
+Requirements:
+
+- Should be easy to update (e.g. dependency bot should not complain about a dozen things for you to update every other day)
```

Comment:
perhaps this could be generalized to low end user maintenance requirement? I suspect that if they upgrade their versions of goreleaser or other tooling they'd also have to interact with changing their configurations, which would fall under the same level of toil.